labeled_data/projects/skywalking.yml

```diff
@@ -0,0 +1,34 @@
+name: Skywalking
```

Comment:
Is this case sensitive? SkyWalking is usually either `SkyWalking` or `skywalking`.

Reply:
Got it.